carmajava/launch/guidance.launch

```diff
@@ -63,6 +63,18 @@
 
   <remap from="system_alert" to="/system_alert"/>
 
+  <remap from="/base/lane_waypoints_raw" to="base/lane_waypoints_raw"/>       
+  <remap from="/lane_waypoints_array" to="lane_waypoints_array"/>
+  <remap from="/traffic_waypoints_array" to="traffic_waypoints_array"/>
+  <remap from="/red_waypoints_array" to="red_waypoints_array"/>
+  <remap from="/green_waypoints_array" to="green_waypoints_array"/>
+  <remap from="trajectory" to="plan_trajectory"/>
+  <!-- TODO use arguments for top level namespace -->
+  <remap from="trajectory_plan" to="/guidance/pure_pursuit/trajectory"/>
+  <remap from="current_pose" to="/localization/ndt_pose"/>
```

Comment:
Lets modify this to use the LOCZ namespace argument. I don't think we need to do the same for /guidance/pure_pursuit/trajectory as that is the plugin topic which will get set by plugin discovery later anyway.